chaos_genius/views/anomaly_data_view.py

```diff
@@ -77,7 +87,7 @@ def kpi_anomaly_detection(kpi_id):
             # remove from here once updated in frontend
             "base_anomaly_id": kpi_id,
         }
-        data["chart_data"]["title"] = kpi_info["name"]
```

Comment:
Why was this removed?

Reply:
Because it's redundant. Both the` get_overall_data()` and `get_dq_and_subdim_data()` functions set title value. 
Must have been an oversight initially.

Reply:
Ah ok